src/component/footer.js

```diff
@@ -0,0 +1,23 @@
+import styles from '@/styles/footer.module.css';
+
+function Footer() {
```

Comment:
Nothing is wrong about the default export here. But have a look on this: https://dev.to/jeetvora331/different-types-of-export-in-react-21p8#:~:text=Use%20default%20export%20when%20you,exports%20in%20the%20same%20file.
To get to know the different types of export.